2-mandatory/8-password-validator.js

```diff
@@ -23,7 +23,31 @@ PasswordValidationResult=  [false, false, false, false, true]
 
 */
 
-function validatePasswords(passwords) {}
+function validatePasswords(passwords) {
+  let previousPasswords = [];
+  return passwords.map((password) => {
+    if (password.length < 5) {
+      return false;
+    }
+    if (!containsUppercaseLetter(password)) {
```

Comment:
You have multiple if statements that will do the same (return false;). How would you optimize this code?